tests/Parsers/StubParser.php

```diff
@@ -61,8 +68,26 @@ public static function processStubs(NodeVisitorAbstract $visitor, callable $file
             $traverser = new NodeTraverser();
             $traverser->addVisitor(new ParentConnector());
             $traverser->addVisitor($nameResolver);
-            $traverser->addVisitor($visitor);
+            $coreStubDirectories = PhpCoreStubsProvider::getCoreStubsDirectories();
+            if (self::stubBelongsToCore($file, $coreStubDirectories)){
+                $coreStubVisitor = new CoreStubASTVisitor(self::$stubs);
```

Comment:
Let's create `CoreStubASTVisitor` in `getPhpStormStubs` and pass it to the function.
This will eliminate the static variable and provide a bit of optimization (instance creation inside foreach).

Reply:
Yeah, I thought about that, but `processStubs` is also used in `MetaExpectedArgumentsCollector` and I've decided not to change to much but just create visitor locally.